### DISCUSSION TITLE
How do I make components in NextJS?

### DISCUSSION THREAD

**Discussion by hbhutta** [vercel/next.js#57625] [Help]
### Summary

I was just using ReactJS until very recently. I learned that in the App router, and subfolder is considered a route. So if I make a `components` subfolder for example, it would be considered a route. I know how to write react components, but how do I organize my project structure to include components?

**Comment by hbhutta** [comment]
Never mind... I figured it out! I just needed to create a folder outside of the Pages or App router. Sorry for the stupid question!

**Comment by jonasermert** [comment]
/src
  /components
    Button.js
    Header.js
    Footer.js
    ...
  /containers
    Home.js
    About.js
    ...
  /pages
    HomePage.js
    AboutPage.js
    ...
  /services
    api.js
    authService.js
    ...
  /utils
    helpers.js
    constants.js
    ...
  /styles
    main.css
    variables.css
    ...
  App.js
  index.js
